lib/client.js

```diff
@@ -324,6 +401,14 @@ Client.prototype = {
   },
 
   context: function() {
+    if (this._instanceGuid) {
```

Comment:
whoops... this line will break context for existing apps... pushing a commit to fix